Doc/library/urllib.parse.rst

```diff
@@ -158,6 +158,10 @@ or on combining URL components into a URL string.
    percent-encoded sequences into Unicode characters, as accepted by the
    :meth:`bytes.decode` method.
 
+   The optional argument *max_num_fields* is a flag for the maximum number of
+   fields to read. If set, then throws a :exc:`ValueError` if there are more
+   than *max_num_fields* fields read. Default is *None*.
```

Comment:
"Default is *None*." IMHO it's redundant with the function prototype and can be removed.